package/gluon-web-model/Makefile

```diff
@@ -9,4 +9,17 @@ define Package/gluon-web-model
   DEPENDS:=+gluon-web
 endef
 
+PKG_CONFIG_DEPENDS += CONFIG_GLUON_MINIFY
+
+define Package/gluon-web-model/install
+	$(Gluon/Build/Install)
+	$(INSTALL_DIR) $(1)/lib/gluon/web/www/static/
+
+ifdef CONFIG_GLUON_MINIFY
+	$(CP) ./javascript/gluon-web-model.min.js $(1)/lib/gluon/web/www/static/gluon-web-model.js
```

Comment:
As these aren't copying recursively, they should use `$(INSTALL_DATA)` instead of `$(CP)` (including the one you didn't change in `gluon-status-page`). The rest LGTM.

Reply:
resolved in 6c0f22445651647a5cd2e7be08e4c7ef6c02e42b, 389314f58a202b36194eec78b084070399025da6, 0ec32078303cb3843408754502529422b9ce5408